sxg_rs/src/process_html.rs

```diff
@@ -240,7 +236,7 @@ mod tests {
                 <script data-issxg-var></script>"
             ),
             "<meta http-equiv=content-type content=\"text/html;charset=&quot;utf-8&quot;\">\
-            <script data-issxg-var></script>",
+            <script data-issxg-var>window.isSXG=true</script>",
```

Comment:
Thanks for the optimization. Please also update the comments at line 228-231.

Reply:
Comment edited. Thanks.